packages/gatsby-recipes/src/renderer/render.js

```diff
@@ -227,18 +215,7 @@ const render = async (recipe, cb, inputs = {}, isApply) => {
 
     RecipesReconciler.render(recipeWithWrapper, plan)
 
-    if (errors.length) {
```

Comment:
removed generic error handling for what we actually do have support for right now namely validation.